magda-registry-api/src/main/scala/au/csiro/data61/magda/registry/RecordsService.scala

```diff
@@ -97,6 +130,33 @@ class RecordsService(config: Config, webHookActor: ActorRef, authClient: AuthApi
     }
   }
 
+  @Path("/")
+  @ApiOperation(value = "Trim by source tag", notes = "Trims records with the provided source that DON'T have the supplied source tag",
+    nickname = "trimBySourceTag", httpMethod = "DELETE", response = classOf[MultipleDeleteResult])
+  @ApiImplicitParams(Array(
+    new ApiImplicitParam(name = "sourceTag", required = true, dataType = "string", paramType = "query", value = "Source tag of the records to PRESERVE."),
```

Comment:
Calling this `sourceTagToPreserve` would greatly reduce the chance of error, I think.

Reply:
Yeah good call 👍